runtime/src/main/kotlin/kotlin/native/internal/GC.kt

```diff
@@ -63,9 +63,36 @@ object GC {
         get() = getThreshold()
         set(value) = setThreshold(value)
 
+    /**
+     * GC allocation threshold, controlling how many bytes allocated since last
+     * collection will trigger new GC.
+     */
+    var thresholdAllocations: Long
+        get() = getThresholdAllocations()
+        set(value) = setThresholdAllocations(value)
```

Comment:
Not used now as I understood, value isn't changed. Is it for future tuning?

Reply:
It's user API for GC tuning.